app/policies/work_lock_policy.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Authorization policy for Work admins to lock/unlock
+class WorkLockPolicy < ApplicationPolicy
+  def edit?
+    administrator?
```

Comment:
Since edit/update are closely coupled, can you make edit an `alias_rule`?  Then there's no duplication.

Reply:
Yup.  Also changed the similar `WorkOwnerPolicy` to match.